blocks/hotspot/hotspot.js

```diff
@@ -0,0 +1,53 @@
+export default function decorate(block) {
+  [...block.children].forEach((row, r) => {
+    if (r > 0) {
+      const content = [...row.children][0].textContent.trim();
+      const isImage = content.endsWith('.jpg') || content.endsWith('.png') || content.endsWith('.gif') || content.endsWith('.jpeg');
+      const isVideo = content.endsWith('.mp4') || content.endsWith('.webm') || content.endsWith('play') || content.endsWith('content/'); // Adjust condition as needed
+      const isText = !isImage && !isVideo; // Assuming if it's neither image nor video, it's text
+
+      const nexticondiv = document.createElement('div');
+      nexticondiv.classList.add('hotspot'); // Added class for CSS targeting
+      nexticondiv.style.left = [...row.children][1].textContent;
+      nexticondiv.style.top = [...row.children][2].textContent;
+      nexticondiv.setAttribute('data', content);
+
+      // Create content display element
+      const contentContainer = document.createElement('div');
+      contentContainer.classList.add('hotspot-content');
+
+      if (isImage) {
```

Comment:
Maybe instead of explicitly adding markup for image, video etc, you could load a fragment and the fragment can contain anything. It'd be more generic. It may be a bit slow though.

Reply:
this is optional btw, and for now we can just support fixed hotspot content.